### ISSUE TITLE
Bitcore wallet service down

### ISSUE THREAD

**Issue by unixfox** [cobit-wallet/cobit#5]
First of all I really appreciate this project because I wanted to keep using Copay on the main chain (not on the segwit2x chain).

Secondly I wanted to use the Bitcore wallet service url provided on the README but it seems to not working (return a 502 HTTP error). Is the server not ready yet?

**Comment by shesek** [comment]
Yes, we're still in the process of setting everything up. Should be ready soon, I'll ping you here when it is. Thanks!
